.github/workflows/ci.yml

```diff
@@ -125,15 +125,11 @@ jobs:
           - os: ubuntu-latest
             arch: amd64
           - os: macos-latest
-            arch: amd64
+            arch: arm64
```

Comment:
This changes the macos run to ARM64. But our build matrix says AMD64 for macos. Shouldn't we have an ubuntu entry with ARM instead?

Reply:
https://github.com/open-telemetry/opentelemetry-go/pull/5575/commits/e67fdcb6c011762357180d389fdbcecefa14dafa

Notice that before macos-14 was latest then the processor arch was amd64. Now it is actually arm64.

Reply:
The alternative could be to keep actuated as it runs on Linux.